package.json

```diff
@@ -39,7 +39,8 @@
     "read": "1.0.7",
     "string-length": "1.0.1",
     "through2": "2.0.1",
-    "tmp": "0.0.28"
+    "tmp": "0.0.28",
+    "update-notifier": "^2.2.0"
```

Comment:
I'll probably go through the [deps](https://david-dm.org/zapier/zapier-platform-cli) and try to update as widely as I can before the 2.0 release.